pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>mate-academy</groupId>
-    <artifactId>builder</artifactId>
+    <artifactId>oop-example</artifactId>
```

Comment:
why have you changed the artifactId?

Reply:
I have an issue with Maven's "default-compile error" and simple copy pow.xml from another project. Now the issue is fixed.